### ISSUE TITLE
Normalize casing of enum variants

### ISSUE THREAD

**Issue by cburgdorf** [ethereum/trinity#1195]
### What is wrong?

We currently use all these different forms of enum casing when we'd ideally be only using a single form.

### Snake Case

```
class Thingy(Enum):
    some_thing = auto()
    other_thing = auto()
```

### Capitalized Snake Case

```
class Thingy(Enum):
    SOME_THING = auto()
    OTHER_THING = auto()
```

### Pascal Case

```
class Thingy(Enum):
    SomeThing = auto()
    OtherThing = auto()
```

### How can it be fixed

1. Decide on casing
2. Correct all places that derive from `Enum`
3. Put the rule into our style guide

**Comment by cburgdorf** [comment]
My vote is on the third option (PascalCase). @ethereum/snake-charmers

**Comment by pipermerriam** [comment]
I'm in favor of still allowing the variant of all caps for the enum values in cases where we mostly only handle the values (as opposed to code that actually instantiates instances of the enum).  It's nice to be able to treat the enum values as constants.

**Comment by cburgdorf** [comment]
@pipermerriam maybe that's a good argument to just settle with `CAPITALIZED_SNAKECASE` then? I think it's good if all follow the same style without exceptions.

**Comment by davesque** [comment]
Yeah, I don't mind the capitalized snake case style.  I recently ran into this problem when trying to write some code in Rust.  The official preferred style in Rust is PascalCase.  However, there are already lots of things that are already PascalCase and having enum variants the same way doesn't distinguish them visually very much.  So I opted to use CAP_SNAKE instead, especially since you can do `use Enum::*;` in Rust.  In that case, the CAP_SNAKE case makes it more obvious where particular identifiers are coming from.

**Comment by wschwab** [comment]
Is this open? I'd like to give it a shot. (I assume there's consensus on CAP_SNAKE.)

**Comment by pipermerriam** [comment]
Go for it

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by cburgdorf** [comment]
This was addressed in #1219
